fedbiomed/common/training_plans/_torchnn.py

```diff
@@ -223,6 +224,14 @@ def optimizer_args(self) -> Dict:
         self.update_optimizer_args()  # update `optimizer_args` (eg after training)
         return self._optimizer_args
 
+    def loader_args(self) -> Dict[str, Any]:
```

Comment:
I agree that it should go into `BaseTrainingPlan`. I didn't do it for "symmetry", because the other similar methods are duplicated in the framework-specific plans. 
But I'll put it in the `BaseTrainingPlan`, and hopefully it will set the example for the other methods ;)